GASocialLogin/Apple/GAAppleLoginService.swift

```diff
@@ -0,0 +1,233 @@
+//
+//  GAAppleLoginService.swift
+//  GASocialLogin
+//
+//  Created by arkadi yoskovitz on 07/07/2020.
+//
+
+import Foundation
+import AuthenticationServices
+
+@available(iOS 13.0, *)
+public typealias GAAppleUserCredential  = ASAuthorizationAppleIDCredential
+public typealias GAAuthorizationKeys    = ASAuthorization.Scope
```

Comment:
this should have a separate @ available marking

Reply:
done